From 19ffef8e60a3cb90b41043ebda3063355896b6a7 Mon Sep 17 00:00:00 2001
From: Michal Privoznik <mprivozn@redhat.com>
Date: Thu, 8 Aug 2013 11:49:25 +0200
Subject: [PATCH 03/48] libvirt-cim.spec.in: Don't install open-pegasus'
 specific providers

https://bugzilla.redhat.com/show_bug.cgi?id=859122

libvirt-cim installs open-pegasus' specific providers in sblim-sfcb
repository staging directory and causing problem when creating sfcb
repository.

Signed-off-by: John Ferlan <jferlan@redhat.com>
---
 libvirt-cim.spec.in | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/libvirt-cim.spec.in b/libvirt-cim.spec.in
index 7b6221e..a027246 100644
--- a/libvirt-cim.spec.in
+++ b/libvirt-cim.spec.in
@@ -133,9 +133,6 @@ then
         -n root/interop \
         -r %{INTEROP_REG} -m %{INTEROP_MOF} -v >/dev/null 2>&1 || true
 %{_datadir}/%{name}/provider-register.sh -t sfcb \
-        -n root/PG_InterOp \
-        -r %{PGINTEROP_REG} -m %{PGINTEROP_MOF} -v >/dev/null 2>&1 || true
-%{_datadir}/%{name}/provider-register.sh -t sfcb \
         -n root/cimv2\
         -r %{CIMV2_REG} -m %{CIMV2_MOF} -v >/dev/null 2>&1 || true
 fi
@@ -165,9 +162,6 @@ then
 	-n root/interop \
 	-r %{INTEROP_REG} -m %{INTEROP_MOF} >/dev/null 2>&1 || true
 %{_datadir}/%{name}/provider-register.sh -d -t sfcb \
-	-n root/PG_InterOp \
-	-r %{PGINTEROP_REG} -m %{PGINTEROP_MOF} >/dev/null 2>&1 || true
-%{_datadir}/%{name}/provider-register.sh -d -t sfcb \
 	-n root/cimv2 \
 	-r %{CIMV2_REG} -m %{CIMV2_MOF} >/dev/null 2>&1 || true
 fi
-- 
1.8.5.3

